rfcs/rfc-004-tictactoe-program.md

```diff
@@ -0,0 +1,76 @@
+
+Two players want to play tic-tac-toe with each other on Solana.
+
+The tic-tac-toe program has already been provisioned on the network, and the
+program author has advertised the following information to potential gamers:
+* `tictactoe_publickey` - the program's public key
+* `tictactoe_gamestate_size` - the number of bytes needs to maintain the game state
+
+The game state is a well-documented data structure consisting of:
+- Player 1 public key
+- Player 2 public key
+- Game status. A 8bit value where:
+  * 0 = game uninitialized
+  * 1 = Player 1's turn
+  * 2 = Player 2's turn
+  * 3 = Player 1 won
+  * 4 = Player 2 won
+- Current board configuration.  A 3x3 character array containing the values '\0', 'X' or 'O'
+
+### Game Setup
+
+1. Two players want to start a game.  Player 2 sends Player 1 their public key,
+`player2_publickey`, off chain (IM, email, ...).
```

Comment:
No comma, and off-chain, and "etc." instead of "..."

Reply:
✔️